e2e/result-cache-8/CustomRule.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace TestResultCache8;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+
+class CustomRule implements Rule
```

Comment:
Your custom rule should be part of your analysed paths. If you fix that I don't think you'll have this problem anymore.

Reply:
> Result cache not used because extension file /var/www/e2e/result-cache-8/src/CustomRule.php hash does not match.

👍 nice thanks a lot for the hint